pkgs/shells/nix-bash-completions/default.nix

```diff
@@ -11,9 +11,21 @@ stdenv.mkDerivation rec {
     sha256 = "093rla6wwx51fclh7xm8qlssx70hj0fj23r59qalaaxf7fdzg2hf";
   };
 
+  # To enable lazy loading via. bash-completion we need a symlink to the script
+  # from every command name.
   installPhase = ''
-    mkdir -p $out/share/bash-completion/completions
-    cp _nix $out/share/bash-completion/completions
+    commands=(nix-build nix-shell nix-env nix-store
+        nix-channel nix-copy-closure nix-hash nix-collect-garbage
+        nix-instantiate nix-install-package nix-prefetch-url nix-push
+        nixos-rebuild nixos-install nixos-version
+        nixos-container nixos-generate-config nixos-build-vms
+        nixops)
+    for c in ''${commands[*]}; do
+      ln -s _nix _$c
```

Comment:
Completions do not start with `_`, see `ls /run/current-system/sw/share/bash-completion/completions`.

Reply:
The lazy loader looks for things starting with `_` too: https://github.com/scop/bash-completion/blob/master/bash_completion#L2032

Reply:
Ah, they may start with `_` (https://github.com/scop/bash-completion/blob/master/bash_completion#L2032), but on my system none does.

Reply:
But yes, it's a bit arbitrary that I've used `_`